napalm_nxos/nxos.py

```diff
@@ -241,6 +241,46 @@ def get_lldp_neighbors(self):
 
         return results
 
+
+    def get_bgp_neighbors(self):
+        cmd = 'show bgp sessions vrf all'
+        tbl = self._get_command_table(cmd, 'TABLE_vrf', 'ROW_vrf')
+
+        ret = {}
+        for el in tbl:
```

Comment:
I will go through all the code next week or in about two weeks and will fix all these kind of things (also from the code I proposed myself a couple of months ago). For the moment, could you please replace all these variables ```el```, ```p``` etc with something more human readable and a minimum of three characters. Thanks!